R/pricesensitivitymeter.R

```diff
@@ -70,11 +85,21 @@ PriceSensitivityMeter <- function(x,
                                   y.title.font.color = global.font.color,
                                   y.tick.font.family = global.font.family,
                                   y.tick.font.color = global.font.color,
+                                  y2.title.font.family = global.font.family,
+                                  y2.title.font.color = global.font.color,
+                                  y2.tick.font.family = global.font.family,
+                                  y2.tick.font.color = global.font.color,
+                                  y2.bounds.minimum = NULL,
```

Comment:
`x.bounds.minimum`, `x.bounds.maximum`, `y.bounds.minimum`, `y.bounds.maximum` are not referenced in this file. Where they forgotten about when this feature was created, or am I missing something?

Reply:
No, they are inherited from `flipStandardCharts::Line` (see line 3). But the line chart doesn't have secondary y-axis

Reply:
I thought `@inheritParams` was just for documentation? How about the code where these parameters are passed to `Line`?

Reply:
It calls `Line` in line 279

Reply:
Ah OK I see the ellipsis parameters in the call to `Line`. Somehow it wasn't placed at the end.